### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by darlanmendonca**
@alencar13 @mauyr @grmelo @bidonmegale 

agree with readme? suggestions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,84 @@
...
-- HTML5 (custom elements)
-- Angular (directive)
-- React (component)
-- Vue (component)
 
-The target is implement a webcomponent using only javascript, and easiliy offer their to any framework.
+## architecture
 
-### Demo
+The core of each component, is a ES6 Class, and for each framework we create a respective component (directive in angular, component in react, ...). So, the core is just one, the ES6 Classs, and we move their to frameworks, respecting the architecture of each one of them.
```
Comment (alencar13) [inline README.md]: Just fix the word "ES6 Classs" to "ES6 Class".

**Comment by alencar13** [comment]
It's perfect! :ok_hand:  Objective and Effective

**Comment by darlanmendonca** [comment]
@alencar13 I protect branch master to only accept merges, by reviewers, and when a commit occur, dismiss review checked :D

pls, approve again, using github interface

**Comment by mauyr** [comment]
:+1:

**Review by mauyr — APPROVED** [review]
:+1:

**Review by alencar13 — COMMENTED** [review]
Approved.

**Pull request merged**